tendrl/commons/objects/definition/master.yaml

```diff
@@ -384,60 +351,6 @@ namespace.tendrl:
       enabled: true
       value: nodes/$NodeContext.node_id/Os
       help: "OS"
-    Package:
```

Comment:
@shtripat Not required, just searched, please remove code (py) too with definitions